.github/ISSUE_TEMPLATE/0-bug-report.yaml

```diff
@@ -0,0 +1,311 @@
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        Please try to give your issue a good title. Try using the product-area where you are having an issue and a brief description of the problem. Like this: 
+        - `Dashboards: Template Variables break when I do X` or 
+        - `Alerting: message templating plus Slack channel breaks when I do X`
+  - type: markdown
+    attributes:
+      value: |
+        **HINT:** Have you tried [searching](https://github.com/grafana/grafana/issues) for similar issues? Duplicate issues are common.
+
+        **Are you reporting a security vulnerability?** [Submit it here instead](https://github.com/grafana/grafana/security/policy).
+  - type: markdown
+    attributes:
+      value: |
+       #
+  - type: textarea
+    id: bug-describe
+    attributes:
+      label: |
+        
+        # 1 / 3: What went wrong?
+        
+      description: |
+        #
+        Describe your bug. What happened? What did you expect to happen?
+
+        **Hot Tip:** Record your screen and add it here as a gif. 
+      placeholder: Tell us what you see!
+      value: |
+        **What happened**:
+
+          - 
+
+        **What did you expect to happen**:
+
+          - 
+
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+       #
+  - type: textarea
+    id: bug-repro
+    attributes:
+      label: |
+        
+        # 2 / 3: How do we reproduce it?
+        
+      description: |
+        #
+
+        Whenever possible, please provide **detailed** steps for reproducing your bug.
+
+        **This is very helpful info** 
+      placeholder: "Step 1:..."
+      value: |
+        **Step 1**:
+        
+        - Open Grafana and do X 
+
+        **Step 2**:
+
+        - Now click button Y
+
+        **Step 3**:
+
+        - Wait for the browser to crash. Error message says: "Error..."
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+       #
+  - type: dropdown
+    id: gf-version
+    attributes:
+      label: |
+        
+        # 3 / 3: What Grafana version are you using?
+        
+      description: |
+        #
+        - You can find your Grafana version:
+          - inside the UI [here](https://share.cleanshot.com/wVTvrKhy), [here](https://share.cleanshot.com/5qg3RKF1), or [here](https://share.cleanshot.com/988MQZz1), depending on your platform and access level.
+          - using the Grafana HTTP API's `health` endpoint [like this](https://share.cleanshot.com/H7gcj2Lj): `curl your-grafana-url/api/health` 
+          - or running the `grafana-server -v` command from Grafana's root directory [like this](https://share.cleanshot.com/LHXfmtZx): `./bin/grafana-server -v`
+      multiple: false
+      options:
+        - I don't know my version
+        - Whatever Grafana Cloud is running
+        - Building from `main`
+        - 9.5.0
```

Comment:
This should probably be generated from our releases page or a free text form?

Reply:
I think we can start with a free text form. Looks like a pain to maintain right now :)

Reply:
agree. system right now is manual and brittle. free-text probably best place to start